presto-main/src/main/java/io/prestosql/operator/ExchangeClientConfig.java

```diff
@@ -36,6 +36,8 @@
     private int clientThreads = 25;
     private int pageBufferClientMaxCallbackThreads = 25;
     private boolean acknowledgePages = true;
+    private int maxRetryCount = 10;
+    private boolean timeoutFailureDetection = true;
```

Comment:
**Suggestion:**
make failure detectors as profile; such as (Timeout, Gossip, heartBeat etc) and define one of the profile as _**default**_
each profile can have there specific configs if need.

Check similar thing done for fileSystem (local, hdfs profiles each have different configs)